site/tool_configuration.md

```diff
@@ -1,10 +1,26 @@
 ## Tools Configuration
+ - [Tool Property File](#tool-properties)
  - [Model Filters](#model-filters)
  - [Target Environments](config/target_env.md)
  - [Type Definitions](#domain-type-definitions)
  - [Variable Injection](variable_injection.md)
  - [Custom Configuration](#custom-configuration)
 
+ ### Tool Properties
```

Comment:
Is there a reason why you used the title, "Tool Property File" in the page TOC versus the heading "Tool Properties" in the section? If not, I suggest that you make them the same.